docs/kpod-push.1.md

```diff
@@ -84,7 +88,7 @@ This example extracts the imageID image to a local directory in docker format.
 
 This example extracts the imageID image to a local directory in oci format.
 
- `# kpod push imageID oci:/path/to/layout`
+ `# kpod push imageID oci-archive:/path/to/layout:image:tag`
 
 This example extracts the imageID image to a container registry named registry.example.com
```

Comment:
Can you add a push example with the --authfile param?

Reply:
fixed